packages/twenty-front/src/modules/app/components/AppRouter.tsx

```diff
@@ -16,12 +18,17 @@ export const AppRouter = () => {
 
   const isAdminPageEnabled = currentUser?.canImpersonate;
 
+  const isPermissionsEnabled = useIsFeatureEnabled(
+    FeatureFlagKey.IsPermissionsEnabled,
```

Comment:
I feel we will need pore granularity on the FeatureFlags in the feature but LGTM!